nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPath.java

```diff
@@ -255,6 +256,55 @@ public static AccessPath getAccessPathForNodeWithMapGet(Node node, @Nullable Typ
     }
   }
 
+  /**
+   * Gets corresponding AccessPath for a class field element.
+   *
+   * @param fieldElement The class field element.
+   * @param nodeReceiver The receiver as a {@link Node}.
+   * @return corresponding AccessPath.
+   */
+  public static AccessPath fromFieldAccessNode(Element fieldElement, Node nodeReceiver) {
```

Comment:
This method seems to be of limited usefulness. I'd prefer to delete and just have callers use `fromFieldAccess` below, passing the correct receiver.  Then we only add one method to this class